src/towncrier/test/test_check.py

```diff
@@ -148,3 +172,52 @@ def test_none_stdout_encoding_works(self):
 
         self.assertEqual(0, proc.returncode)
         self.assertEqual(b"", stderr)
+
+    def test_first_release(self):
```

Comment:
It would be nice to have a docstring explaining what is the purpose of this test.
Together with a docsting from test_second_release they should help understand the differences between these 2 tests.

We don't have tests for testing code and in order to be able to validate that the test is correct we need a docstring explaining the purpose of each test.

Reply:
Done